test-requirements.txt

```diff
@@ -9,3 +9,6 @@ PyYAML
 yamllint
 nose
 coverage
+git+https://github.com/detiber/molecule@v2_tweaks
+pytest-xdist
```

Comment:
I believe it is good enough for this purpose, it actually  helped uncover a bug with the ansible modules where we weren't using a copy of the kubeconfig.

I think there are some other improvements that could speed up test runs, and if that solves the runtime issue then I will remove the pytest-xdist dependency.